src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -219,10 +248,18 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer using advanced options.
+        /// For usage details, please see <see cref="AveragedPerceptron(BinaryClassificationCatalog.BinaryClassificationTrainers, string, string, string, IClassificationLoss, float, bool, float, int)"/>
```

Comment:
>For usage details [](start = 12, length = 17)

This goes in remarks, so it doesn't show up in the table of contents.

Reply:
+1

---
In reply to: [256223581](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256223581) [](ancestors = 256223581)

Reply:
moved

---
In reply to: [256250971](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256250971) [](ancestors = 256250971,256223581)